nslsii/temperature_controllers.py

```diff
@@ -0,0 +1,81 @@
+from ophyd import DeviceStatus, Device, Component as Cpt, EpicsSignal, Signal
+
+
+class Eurotherm(Device):
+    '''This class is used for integrating with Eurotherm controllers.
+
+    Parameters
+    ----------
+    PV_prefix : str.
+        The PV prefix that is common to the readback and setpoint PV's.
+    tolerance : float, optional.
+        The range of temperature within which it can be considered constant.
+        Default is 1.
+    EquilibriumTime : float, optional.
+        The time (in seconds) that a temperature should be within `tolerance`
+        for it to be considered at equilibrium. Default is 5 seconds.
+    Timeout : float, optional.
+        The maximum time (in seconds) to wait for the temperature to reach
+        equilibrium before raising a TimeOutError. Default is 30 seconds.
+    '''
+
+    def __init__(self, PV_prefix, **kwargs):
+        super().__init__(PV_prefix, **kwargs)
```

Comment:
In beamline profiles this block normally goes after all components are defined. Maybe worth to move it below.

Reply:
I am not sure this is true, at least [here](https://github.com/NSLS-II-SIX/profile_collection/blob/77ee29dcd530aa95c61401efdc94ad7c6c21bba8/startup/21-areadetector.py#L22) it is not. @danielballan  or @tacaswell  do we have a standard regarding if the `__init__` comes before or after `component` definitions? If so I will change this to match the standard.